index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
The formatting is ugly. Please use an automatic formatter, such as [prettier](https://prettier.io/docs/en/install.html)

Reply:
Done.